front/src/components/app.jsx

```diff
@@ -200,7 +200,20 @@ class MainApp extends Component {
   }
 
   render({ user }, {}) {
-    const translationDefinition = user && user.language && user.language === 'fr' ? translationFr : translationEn;
+    let language;
+    if (!user) {
+      // Get language from browser
+      language = (
+        (navigator.languages && navigator.languages[0]) ||
```

Comment:
We already use the navigator language here, so it's a little redundant => 

https://github.com/GladysAssistant/Gladys/blob/master/front/src/utils/getDefaultState.js#L30

I think the user && user.language we use to have is not useful, because I think that user is never empty, is it?

Reply:
You're right. But in default state, we try to compare 'fr-FR' with 'fr'. I will propose something to handle this.